cmake/scripts/linux/ArchSetup.cmake

```diff
@@ -16,10 +16,11 @@ else()
   elseif(CPU MATCHES arm)
     set(ARCH arm)
     set(NEON True)
-    set(NEON_FLAGS "-mfpu=neon -mvectorize-with-neon-quad")
+    set(NEON_FLAGS "-mfpu=neon -funsafe-math-optimizations -mvectorize-with-neon-quad")
   elseif(CPU MATCHES aarch64 OR CPU MATCHES arm64)
     set(ARCH aarch64)
-    set(NEON False)
+    set(NEON True)
+    set(NEON_FLAGS "-march=armv8-a -funsafe-math-optimizations")
```

Comment:
aarch64 is always neon. you dont need these, and you should not override user's specified -march= anyway